docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -9,65 +9,9 @@ include::./attributes.adoc[]
-
-====
-.Example
- 
-If the user name is `Alice` and the password is `secret`, the HTTP authorization header looks authorization as `Authorization: Basic  QWxjZTpzZWNyZXQ=`, where `QWxjZTpzZWNyZXQ=` is a Base64 encoded representation of the `Alice:secret` string.
-====
- 
-The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
-
-Basic Authentication is a well-specified, simple challenge and response scheme that all web browsers and most web servers understand. However, there are a few limitations associated with Basic Authentication, which include:
-
-Credentials are sent as plain text::
-+
---
-It is required to use HTTPS with Basic Authentication to avoid exposing the credentials. However, if the load balancer terminates HTTPS, the risk of exposing credentials as plain text increases when a request is forwarded to Quarkus over HTTP.
-
-Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only, and the credentials are propagated to the next Quarkus endpoint over HTTP.
---
-
-Credentials are sent with each request::
-+
---
-In Basic Authentication, a username and password need to be sent with each request, which increases the risk of credentials being exposed.
---
-
-Application complexity increases::
-+
---
-The Quarkus application needs to ensure itself that usernames, passwords, and roles are managed securely, which increases the application complexity.
---
-
-[[enabling-basic-auth]]
-=== Enabling Basic Authentication
- 
-.Prerequisites
- 
-* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
- 
-.Procedure
- 
-* Enable Basic Authentication by setting the value of `quarkus.http.auth.basic` property to `true`.
-+
-[source,properties]
-----
-quarkus.http.auth.basic=true
-----
- 
-For a Basic Authentication configuration walk-through that uses `JPA`, see the xref:security-getting-started.adoc[Getting Started With Security] guide.
- 
-.Additional resources
- 
-* xref:security.adoc#identity-providers[Security Identity Providers]
-* xref:security-testing.adoc#configuring-user-information[Configuring User Information in application.properties]
+include::security-basic-auth-concept.adoc[leveloffset=+1]
+
+include::security-enabling-basic-auth-tutorial.adoc[leveloffset=+2]
```

Comment:
Is this just to keep some consistency with the existing guide?

I don't expect to have docs include content in this way in the long term.

Reply:
@hmanwani-rh Heena as we discus on chat, these two includes should be both of equal heading level. The procedure is not a subset of that concept.

Reply:
@ebullient To adapt the diataxis framework, we need to create modules for concepts and tutorials, therefore, we added those modules using include in the same file the sections were available before.

Reply:
> @hmanwani-rh Heena as we discus on chat, these two includes should be both of equal heading level. The procedure is not a subset of that concept.

Discussed on chat. Keeping the heading level as it was earlier.

Reply:
> @ebullient To adapt the diataxis framework, we need to create modules for concepts and tutorials, therefore, we added those modules using include in the same file the sections were available before.

Just note this is temporary... eventually the old guide will go, and we'll achieve navigation a different way. We aren't going to shoot for reusable/granular documents as the end game, it confuses navigation and search (where they would be both separate and embedded.. )

Reply:
@hmanwani-rh `security.adoc` refers to this removed section, so that link needs to be updated to point to a new content

Reply:
@sberyozkin Thanks for letting me know. I've updated the links in security.adoc.